phpcs.xml.dist

```diff
@@ -25,15 +25,13 @@
 	<arg value="ps"/>
 	<arg name="extensions" value="php"/>
 
-	<file>.</file>
-
-	<!-- Exclude 3rd party libraries -->
-	<exclude-pattern>./node_modules</exclude-pattern>
-	<exclude-pattern>./vendor</exclude-pattern>
+	<file>./lib</file>
+	<file>./packages</file>
+	<file>./phpunit</file>
```

Comment:
```suggestion
	<file>./phpunit</file>
	<file>./bin</file>
	<file>./post-content.php</file>
```

If I run `composer run lint` in the master branch, I see 60 files. After this PR I see 56 files. 
The `/bin` folder and `post-content.php` are missing

Reply:
> If I run `composer run lint` in the master branch, I see 60 files.
> After this PR I see 56 files.
> The `/bin` folder and `post-content.php` are missing

Good catch! Corrected in 6123772.